src/content/contributing/shipyard/_index.en.md

```diff
@@ -6,7 +6,7 @@ weight: 80
 
 ## Overview
 
-The Shipyard project provides common tooling for creating K8s clusters with [KIND](https://github.com/kubernetes-sigs/kind) (K8s in Docker)
+The Shipyard project provides common tooling for creating K8s clusters with [kind](https://github.com/kubernetes-sigs/kind) (K8s in Docker)
```

Comment:
Not related to this PR, but we also need to change "K8s" to "Kubernetes" across the website.

Reply:
Good idea, I'll send another one to do that. :+1:

Reply:
https://github.com/submariner-io/submariner-website/pull/303